documentation/outlet-control/index.rst

```diff
@@ -0,0 +1,110 @@
+.. _Outlet-control-doc:
+
+**************
+Outlet control
+**************
+
+.. warning:: Outlet control operations will affect connected devices. These operations have the same effect as the corresponding physical actions. It is the responsibility of the user to assess the situation and act accordingly.
+
+Hyperview Outlet Control has been designed to help you manage and control the power state of compatible Smart Power Distribution Units (rPDUs) while maintaining a detailed record of all actions taken by users.
+
+This feature allows you to:
+
+- Turn on compatible rPDUs - Remotely power up individual outlets, ensuring that power is allocated only when and where it is required.
+
+- Turn off compatible rPDUs - Remotely power down individual outlets, allowing you to save energy and reduce operational costs.
+
+- Power cycle compatible rPDUs - Remotely perform power cycling on individual outlets, enabling you to reset unresponsive equipment and troubleshoot potential issues without the need for onsite intervention.
+
+Work Orders maintain a comprehensive paper trail of all control requests made by users, allowing you to review historical actions and ensure accountability for any changes made.
+
+.. note:: Only licensed instances will have relevant features. To confirm if you have an Outlet Control license, check the License page (*Settings → License*, Administrator-only).
+
+==========================
+Supported hardware vendors
+==========================
+The Outlet Control feature is **definition based** meaning that the system is adaptive and support can be added quickly with the right information.
+
+At the time of release, a large number of device models from the following vendors are supported:
+
+- Schneider Electric/APC
+- Avocent
+- Baytech
+- Eaton
+- Enlogic
+- Geist
+- Liebert
+- Panduit
+- Raritan
+- Server Technology
+- Tripp Lite
+
+If you would like Hyperview to add support for a new model or manufacturer please contact Hyperview support.
+
+====================
+Using outlet control
+====================
+For an rPDU, outlet control functions are exposed on the Layout page. Please note the following requirements:
+
+1. A Control license. This enables the Outlet Control, Firmware Management and Rack Security features
+2. A supported rPDU
+3. A configured rPDU. I.e. The rPDU itself is configured to allow outlet control and the appropriate :ref:`control operations<Control-operations-doc>` parameters set. This can be done in bulk or on an individual rPDU basis
+
+.. image:: media/rpdu-layout.png
```

Comment:
For consistancy:
- You could move all the images to a media.rst file.
- Add a :class: border-black to each of the images.

Reply:
Thank! I can add a class attribute. 
In terms of media.rst, I am transitioning to inline images as I find this more readable and easier to manage.

Reply:
added class attribute. thank!